core/src/thread_parker/sgx.rs

```diff
@@ -18,6 +17,18 @@ use std::{
     thread,
     time::Instant,
 };
+use core::sync::atomic::{AtomicBool, Ordering};
+
+
```

Comment:
Run rustfmt to clean up stuff like this.

Reply:
Done